04-onion-routing.md

```diff
@@ -73,7 +73,7 @@ A node:
 
 There are a number of conventions adhered to throughout this document:
 
- - Length: the maximum route length is limited to 20 hops.
+ - Length: the maximum route length is limited to 19 _intermediate nodes_, the _origin node_ and _final node_ do not count toward the limit.
```

Comment:
I think we should keep the "20 hops" wording. The change can instead be an added clarification.

Reply:
Sounds good to me, after all whatever removes the ambiguity will do.